Makefile

```diff
@@ -129,4 +137,21 @@ docs:
 	rm -rf docs/_build/
 	@${ACTIVATE} sphinx-build docs/ docs/_build/
 
+keycloak:
+	@echo "-> Run Keycloak Identity provider"
+	docker run -dit --name scancodeio-keycloak \
+                    --env KEYCLOAK_USER=${KEYCLOAK_USER} \
+                    --env KEYCLOAK_PASSWORD=${KEYCLOAK_PASSWORD} \
+                    --publish ${KEYCLOAK_HTTPS_BIND_PORT}:8443 \
+                    jboss/keycloak
+	@echo "-> Keycloak launched. Please visit https://localhost:${KEYCLOAK_HTTPS_BIND_PORT} and configure realm"
+
+oidc:
+	@echo "-> Configure OIDC settings"
+	@echo "" >> ${ENV_FILE}
+	@echo OIDC_OP_REALM_ENDPOINT=\"https://keycloak:${KEYCLOAK_HTTPS_BIND_PORT}/auth/realms/${KEYCLOAK_REALM_NAME}/\" >> ${ENV_FILE}
```

Comment:
add env for `https://keycloak` so it can be easily configured when having keycloak running on different domain